android/TPSDropDown/testmodule/src/test/java/com/gettipsi/testmodule/ExampleUnitTest.java

```diff
@@ -0,0 +1,15 @@
+package com.gettipsi.testmodule;
```

Comment:
This is the module which contains tests and activity for start, because library hasn't activity